Tasks/JenkinsQueueJobV2/job.ts

```diff
@@ -131,6 +131,14 @@ export class Job {
         this.working = false;
     }
 
+    
+    private RetryConnection(): void {
+        const thisJob: Job = this;
```

Comment:
Why do we need this?
 thisJob.consoleLog -> this.consoleLog?

Reply:
We need this for stopWork and retryCount

Reply:
Why it can't be like 'this.consoleLog' instead?

Reply:
It can be, tried it before and it didn't work, probably due to some faulty code on my side. Now it works fine. Changed everyting to this